libconsensus/rotating_pbft/RotatingPBFTEngine.h

```diff
@@ -80,23 +81,33 @@ class RotatingPBFTEngine : public PBFTEngine
         return m_blockSync->isFarSyncing() && m_locatedInConsensusNodes;
     }
 
+    /// get sealer list
+    dev::h512s consensusList() const override

```

Comment:
Why not return shared_ptr/const &

Reply:
Since `m_chosedSealerList` will be modified when the sealers updated, this function should return the copy of  `m_chosedSealerList` for thread-safe consideration.